docs/concepts/command-handler.md

```diff
@@ -9,5 +9,19 @@ type: markdown
 <h2 class="top">Command Handler</h2> 
 
 Command Handler is an object which receives commands, modifies the state of the application, and generates events if the modification was successful. 
+Here is a code example which shows how a method which handles this kind of messages looks like.
```

Comment:
This code must go under `Task Subscriber`.

Reply:
Should we create a new term entry for the Concepts section and name it `Task Subscriber` then?

Reply:
No, I meant “Event Subscriber”.